contrib/verify-commits/verify-commits.py

```diff
@@ -94,7 +94,12 @@ def main():
     branch = subprocess.check_output([GIT, 'show', '-s', '--format=%H', initial_commit]).decode('utf8').splitlines()[0]
 
     # Iterate through commits
+    commit_number = 1
     while True:
+        # Log a message to prevent Travis from timing out
+        if commit_number % 50 == 0:
+            print("verify-commits: [in-progress] processing commit number {} ({})".format(commit_number, current_commit[:8]))
```

Comment:
If you print anyway, why not print every commit, but make it a log that is only enabled with log level debug?